### PR TITLE
React home not found

### PR DESCRIPTION
**Pull request by jricks86**

### CODE REVIEW

File: src/__tests__/Home.test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        )
+    })
+    it("has a clickable link", () => {
+        render(
+            <BrowserRouter>
+                <Home />
+            </BrowserRouter>,
+        
+        )
+        userEvent.click(screen.getByText("Book today!"))
```
Comment (SunkissedQueen) [inline src/__tests__/Home.test.js:23]: nice job finding different ways to test your UI

File: src/__tests__/NotFound.test.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+//       --------------------------------------------------
+//       link:
+
+//       Name "Let's go home!":
+//       <a
+//         class="btn btn-info text-dark"
+//         href="/"
+//         style="width: 50%;"
+//       />
+
```
Comment (SunkissedQueen) [inline src/__tests__/NotFound.test.js]: Please remove or place in the README. Question is this indicating that the test failed?

File: src/components/Header.js [hunk 3]
```diff
@@ -7,33 +7,28 @@ import {
...
-  DropdownItem,
-  NavbarText,
 } from 'reactstrap';
 
-function Header(args) {
+function Header() {
   const [isOpen, setIsOpen] = React.useState(false);
 
   return (
 <div style={{
```
Comment (SunkissedQueen) [inline src/components/Header.js]: Looks like the indentation is off. Please check the indentation on all your files.

**Review by SunkissedQueen — CHANGES_REQUESTED** [review]
Good job planning and having a space for the whole team to grow as developers. Those pics are cute.
Concerns with this PR:

1. I see an error code message on one of the tests. Did that test fail? Please only request a review for working code. If you need help with blockers, let's talk through it.
2. Indentation. Please go through and check that all your files are properly nested.

File: src/__tests__/NotFound.test.js (hunk 2 above)
Comment (cherrythefool) [reply src/__tests__/NotFound.test.js]: Hi! It's passing as of the PR, dropped it in there while we were troubleshooting but forgot to take this out

File: src/components/Header.js (hunk 3 above)
Comment (cherrythefool) [reply src/components/Header.js]: thank you! in progress

File: src/__tests__/NotFound.test.js (hunk 2 above)
Comment (cherrythefool) [reply src/__tests__/NotFound.test.js]: fixed

File: src/components/Header.js (hunk 3 above)
Comment (cherrythefool) [reply src/components/Header.js]: standardized to four spaces, fixed~

**Review by SunkissedQueen — APPROVED** [review]

**Pull request merged**